src/pages/{storyblokEntry.full_slug}.tsx

```diff
@@ -15,20 +16,34 @@ const StoryblokEntry: React.FC<PageProps<DataProps>> = ({
 }) => {
   let story = data.storyblokEntry;
   story = useStoryblokState(story);
+  const blok = story.content;
 
   return (
     <Layout>
       {/* Place holder hero below - going to extract into component */}
       <Hero heading={story.name} />
-      <div {...storyblokEditable(story.content)}>
-        <CreateBloks blokSection={story.content.content} />
-      </div>
+      <CreateBloks blokSection={blok.hero} />
+      <CreateBloks blokSection={blok.content} />
     </Layout>
   );
 };
 
 export default StoryblokEntry;
 
+export const Head = ({ data }) => {
+  let story = data.storyblokEntry;
+  story = useStoryblokState(story);
+  const blok = story.content;
+
+  return (
+    <PageHead
+      title={blok.title || story.name}
+      heroImage={blok.heroImage || blok.hero.image}
+      seo={blok.seo}
+    />
+  );
+};
```

Comment:
This is the new way of handling html head elements in Gatsby 5. I'm asking a question on Discord whether it's better to put the font stuff inside `PageHead`, or in the `gatsby-ssr`. Their official doc still says gatsby-ssr, but their document is all over the place at the moment.